app/assets/stylesheets/application.scss

```diff
@@ -47,6 +47,11 @@ body {
   min-height: 100vh;
   flex-direction: column;
 }
+body.show {
```

Comment:
@eanders - this fixes the footer not sticking to the bottom because when `show` gets added to the body class, it triggers `display: block` in the Bootstrap code which has a declaration of `important` and overrides the `flex: display` css being used to shove the footer to the bottom. Bug only happens on show pages.